src/reportSources/legacyFflogs/__tests__/__snapshots__/eventAdapter.ts.snap

```diff
@@ -39,7 +39,7 @@ Array [
 exports[`Event adapter individual events adapts applydebuff 1`] = `
 Array [
   Object {
-    "action": 2258,
+    "action": 2248,
```

Comment:
No clue what this file is; do you know what the change here means?

Reply:
It's some unit test file that broke now that the old Trick Attack raid buff status was fully gone from the data files. I had to switch it to the Mug raid buff status to get it to pass.